### PR TITLE
Adding vcfsim

### PR DESCRIPTION
**Pull request by Pie115**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Pie115** [comment]
I agree to be a maintainer

**Comment by Pie115** [comment]
@conda-forge/help-python ready for review!

**Comment by Pie115** [comment]
@conda-forge/staged-recipes this is ready for review.

File: recipes/vcfsim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  url: https://github.com/Pie115/VCFSimulator-SamukLab/archive/{{ version }}.tar.gz
+  sha256: 3930958c2e52f62e5dd9e685307ff03e5387ae80158339806152da7f7aaa57ac
+
+build:
+  entry_points:
+    - vcfsim=vcfsim.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True
```
Comment (mfansler) [inline recipes/vcfsim/meta.yaml:18]: Nothing is being built.

File: recipes/vcfsim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{% set name = "VCFSimulator-SamukLab" %}
+{% set version = "1.0.5.alpba" %}
```
Comment (mfansler) [inline recipes/vcfsim/meta.yaml:2]: This looks like a typo.

File: recipes/vcfsim/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  entry_points:
+    - vcfsim=vcfsim.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (mfansler) [inline recipes/vcfsim/meta.yaml:22]: `noarch` cannot also involve compilation. The package repository appears to be consistent with `noarch`, so the compilation components should likely be omitted.

**Review by mfansler — CHANGES_REQUESTED** [review]
Please consider [submitting to Bioconda](https://bioconda.github.io/contributor/guidelines.html) instead. This appears to be a prerelease of bioinformatics research software that is intended for `noarch` builds. That makes Bioconda seem like a better fit.

Otherwise, is there a reason why this cannot be released to PyPI first?

File: recipes/vcfsim/meta.yaml (hunk 3 above)
Comment (Pie115) [reply recipes/vcfsim/meta.yaml:22]: Thank you for the help and suggestions, I will look into this ASAP.

**Pull request closed without merging**